buildscripts/build-tests.cmd

```diff
@@ -30,7 +30,7 @@ exit /b %ERRORLEVEL%
 if defined __SkipTests exit /b 0
 
 pushd "%__ProjectDir%\tests"
-call "runtest.cmd" %__BuildType% %__BuildArch% /dotnetclipath %__DotNetCliPath%
+call "runtest.cmd" %__BuildType% %__BuildArch% /dotnetclipath %__DotNetCliPath% /test HelloWasm
```

Comment:
Please remove this

Reply:
oops, I think i must have accidentally done commit -am